packages/tabsheet/src/vaadin-tabsheet.js

```diff
@@ -3,7 +3,269 @@
  * Copyright (c) 2022 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
+import '@vaadin/tabs/vaadin-tab.js';
```

Comment:
Let's change this to `src` import:

```suggestion
import '@vaadin/tabs/src/vaadin-tab.js';
```